connectivity/pxp_reporter_makefile/build/Makefile

```diff
@@ -0,0 +1,397 @@
+# This target will get run if just `make` is used. Be sure to set the BUILD_CONFIG to one of BUILD_CONFIG_TYPES
+default: all
+
+-include makefile.init
+
+SOURCEROOT ?= ../../../../..
+SDKROOT := $(SOURCEROOT)/sdk
+APP_SOURCE_DIR ?= $(SOURCEROOT)/projects/dk_apps/demos/pxp_reporter_makefile
+CONFIG_SOURCE_DIR ?= $(APP_SOURCE_DIR)/config
+DEVICE := DEVICE_DA1469x
+
+BUILD_CONFIG ?=  # one of BUILD_CONFIG_TYPES below
+BUILD_CONFIG_TYPES := debug_qspi 
+BUILD_CONFIG_TYPES += debug_qspi_suota 
+BUILD_CONFIG_TYPES += debug_ram 
+BUILD_CONFIG_TYPES += release_qspi 
+BUILD_CONFIG_TYPES += release_qspi_suota
+BUILD_CONFIG_TYPES += release_ram
+
+BUILD_OUTPUT_DIR = $(BUILD_CONFIG)
+
+CC := arm-none-eabi-gcc
```

Comment:
In my case, the gcc compiler is not part of my PATH, therefore I made the following changes:
TOOLDIR := /cygdrive/c/DiaSemi/SmartSnippetsStudio2.0.20/GCC/7-2018-q2/
CC = $(TOOLDIR)/bin/arm-none-eabi-gcc.exe
BIN = $(TOOLDIR)/bin/arm-none-eabi-objcopy.exe
SIZE = $(TOOLDIR)/bin/arm-none-eabi-size.exe

Reply:
This could also be used with a different gcc compiler version

Reply:
Variables added in [1ba3fc9](https://github.com/dialog-semiconductor/BLE_SDK10_examples/pull/30/commits/1ba3fc9295fb8640ee6ef1eb723ecbc2f42c2ad1)